test/python/expected/upgrade_validation/expected_dependency.out

```diff
@@ -135,6 +135,8 @@ Function sys.babelfish_get_login_default_db(text)
 Function sys.babelfish_get_microsecs_from_fractsecs(text,numeric)
 Function sys.babelfish_get_monthnum_by_name(text,jsonb)
 Function sys.babelfish_get_name_delimiter_pos(text)
+Function sys.babelfish_get_scope_identity()
+Function sys.babelfish_get_scope_identity_numeric()
```

Comment:
This means we don't have a stored procedure / view directly depending on them. 
We could avoid them by not having them.